libvast/src/system/node.cpp

```diff
@@ -519,7 +518,7 @@ caf::behavior node(node_actor* self, std::string name, path dir,
                    std::chrono::milliseconds shutdown_grace_period) {
   self->state.name = std::move(name);
   self->state.dir = std::move(dir);
-  // Initialize component and command factories.
+  // Initialize component and command factories.,
```

Comment:
Nope 😉

Reply:
I suppose this is what happens if you set `,` as leader. 🙈